### PR TITLE
About Section and Works Section

### PR DESCRIPTION
**Pull request by GhostEsso**
- [ ] I Made sure the content of the "About myself" section is aligned with flexbox.

### CODE REVIEW

**Comment by GhostEsso** [comment]
Thank you for everything, I look forward to your corrections and appreciations

File: index.html [hunk 1]
```diff
@@ -184,6 +184,106 @@ <h2 class="project-title">Tonic</h2>
           </p>
         </div>
         <div class="project-tag">
+          <ul class="project-skills flex">
```
Comment (RaoAkif) [inline index.html:187]: I found that you are using `background-color:#fff` on
<section class="project-container">
But
this section does not have any background itself.
The background color of this container comes from the background color of `<body>` and that color is `#F7F7F9`
So when I applied both colors: I found this:

image





### ⭕ I would also like to share the current comparison with the above guidance on what can be done.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| image| image|

File: index.html [hunk 2]
```diff
@@ -184,6 +184,106 @@ <h2 class="project-title">Tonic</h2>
...
+            <li>html</li>
```
Comment (RaoAkif) [inline index.html:188]: You are using `font-size:40px` on the following div. 
`<h2 class="project-title">Tonic</h2>`

The font-size being used in FIGMA design is `font-size:32px`
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| image| image|

File: index.html [hunk 3]
```diff
@@ -184,6 +184,106 @@ <h2 class="project-title">Tonic</h2>
...
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+        <button class="see-project-btn" type="button">See Project</button>
+      </div>
+    </div>
+  </section>
+  <section id="about" class="about">
```
Comment (RaoAkif) [inline index.html:197]: There is more space between the title of the section and the top of the section. You can try to approach to a total of `114px` using flex sizing and margin etc.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
|  image| image|

File: index.html [hunk 4]
```diff
@@ -184,6 +184,106 @@ <h2 class="project-title">Tonic</h2>
...
+        <p class="para-text">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <div class="self-container">
+          <p>Let's Connect</p>
+          <div class="list-nav">
+            <ul class="list-items skill-items">
```
Comment (RaoAkif) [inline index.html:210]: - The LinkedIn icon is one image. Kindly download it in this way.
image

- The size of the font icons in Figma design is around `20px`. The width and height change as per the shape of the icon. I am sharing the sizes comparison: Kindly apply that size to get near to the design.

File: index.html [hunk 5]
```diff
@@ -184,6 +184,106 @@ <h2 class="project-title">Tonic</h2>
...
+        <ul>
+          <li>
+            <div class="lang flex column">
+              <p class="lang-text flex">
+                <span class="txt">Language</span><span class="lang-line">></span>
+              </p>
+              <ul class="lang-div flex column">
+                <li class="lang-list flex">
+                  <img src="images/Js.svg" alt="js logo" />
+                  <p class="skill-text">JavaScript</p>
```
Comment (RaoAkif) [inline index.html:250]: You missed this `font-size` This is `15x` as per Figma design.
image

File: index.html [hunk 6]
```diff
@@ -184,6 +184,106 @@ <h2 class="project-title">Tonic</h2>
...
+                <a href="">
+                  <img src="images/Twitter.svg" alt="my social handle images" />
+                </a>
+              </li>
+              <li>
+                <a href="">
+                  <img src="images/Vector.svg" alt="my social handle images"/><img src="images/Vector (1).svg" alt="my social handle images" />
+                </a>
+              </li>
+              <li>
```
Comment (RaoAkif) [inline index.html]: Your design contains these background images. Figma design does not contain such images.
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| image| image|

**Review by RaoAkif — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️

Hi @GhostEsso, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- When you will fix the background color of the works section. Also, keep in mind the color of the first landing section. It would get disturbed.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (GhostEsso) [reply index.html:210]: @RaoAkif on the screenshot, it's a github icon... when i'm trying to download linkedin icon, it downloads it seperatly... how can i download it in correct way please?

Comment (GhostEsso) [reply index.html:210]: @RaoAkif i found, it's okay

**Comment by GhostEsso** [comment]
I corrected recommendations

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @GhostEsso , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlight 
:+1: no linter errors 
:+1: your design is great

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**